Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Traits.adq

```diff
@@ -2028,7 +2028,7 @@
 			"type": "trait",
 			"name": "Combat Reflexes",
 			"reference": "DFA48",
-			"notes": "Never freeze",
+			"reference_highlight": "Combat Reflexes",
```

Comment:
This seems unneeded. I believe the name is already used for this purpose and the reference_highlight field is only needed when the book actually uses something else.

Reply:
Ahh! I didn't fully understand how that field was intended to be used. I'll revert those.